view/theme/frio/css/style.css

```diff
@@ -2531,7 +2522,14 @@ section .profile-match-wrapper {
 	background-color: $link_color;
     border-color: $link_color;
 }
-
 .disabled > a {
 	pointer-events: none;
-}
\ No newline at end of file
+}
+
+/*
+ * some temporary workarounds until this will solved
```

Comment:
It's apparently a whole category, each temporary workaround should have its own `@todo` but I didn't create the current one, so I would be hard-pressed to write the relevant documentation piece.